.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,24 @@
+# Contributions
+
+We welcome contributions in the form of issues and pull requests.  We view the contributions and the process as the same for github and external contributors.
```

Comment:
> We view the contributions and the process as the same for github and external contributors.

Consider removing the sentence. Might be confusing what "github" contributors are vs "external" - overwhelming majority of folks reading will be external. Feels unnecessary and a lot to untangle, especially in the intro.

Otherwise consider removing extra "the" and changing "github" to "internal" or "GitHub":

> We view the contributions and ~the~ process as the same for ~github~ internal and external contributors.